web/src/lib/i18n/en.json

```diff
@@ -67,6 +67,10 @@
     "image_thumbnail_format": "Thumbnail format",
     "image_thumbnail_resolution": "Thumbnail resolution",
     "image_thumbnail_resolution_description": "Used when viewing groups of photos (main timeline, album view, etc.). Higher resolutions can preserve more detail but take longer to encode, have larger file sizes, and can reduce app responsiveness.",
+    "import_faces_metadata": "Import faces from metadata",
+    "import_faces_metadata_description": "Import faces from EXIF/sidecar files",
+    "import_faces_metadata_setting": "ENABLED",
```

Comment:
this probably should be an existing key or translated as something else

Reply:
Hi @jrasm91 I've simplified it a bit (6289e0ebb4266b1a84a7f68c87e823610a351aae) and this is how it looks like:

![metadata-faces-import-preview](https://github.com/user-attachments/assets/5c9c51ee-3a95-45ba-b384-28451c117cf9)